src/components/editor/task-bar/editor-menu.scss

```diff
@@ -0,0 +1,6 @@
+@import "node_modules/bootstrap/scss/functions";
+@import "node_modules/bootstrap/scss/variables";
+
+.dropdown-item {
+  font-size: $font-size-sm;
```

Comment:
Please don't do this! You're overriding this class for the WHOLE app! Use the `small` class instead.

Reply:
And if `small` doesn't do what you want, create a new css class.